cropper/src/main/res/menu/crop_image_menu.xml

```diff
@@ -1,18 +1,22 @@
 <?xml version="1.0" encoding="utf-8"?>
 <menu xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
Now that we have 3 dots menu because the amount of icons the order should be:
`Crop > Shape > Flip > Rotate`